Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM perl:5.22
+
+ENV PERL_MM_USE_DEFAULT=1 PERL_CARTON_PATH=/carton
+
+COPY cpanfile cpanfile.snapshot /metacpan-web/
+WORKDIR /metacpan-web
+
+RUN cpanm App::cpm Carton && \
+    useradd -m metacpan-web -g users && \
+    mkdir /carton && \
+    cpm install -L /carton
+
+COPY . /metacpan-web
```

Comment:
Is there a reason to do this copy rather than just running directly from the repo?